addons/web/static/tests/_framework/mock_server/mock_model.js

```diff
@@ -1274,6 +1274,7 @@ export class Model extends Array {
                     model._onChanges = { ...previous._onChanges };
                     model._order = previous._order;
                     model._parent_name = previous._parent_name;
+                    model._fold_name = previous._fold_name;
```

Comment:
For consistency, please put it in alphabetical order with other properties (i.e. after `_fields`), and do the same for the getter/setter & assignation.

Also you forgot to put it in `get definition()` of the `Model` class (I know this is tedious, but hopefuly we don't have to add properties that often :smile:)

Reply:
Hello 👋 Thanks for the review!
My bad, forgot the alphabetical order :man_facepalming: 

For the declaration of the field `get definition()`, is it not that?
https://github.com/odoo/odoo/pull/188769/files#diff-edcb01873d9c6224e764910b0189bbdff336be6af9abec67296d083165df5760R1272 

Thanks in advance :pray:

Reply:
Indeed, didn't see it, LGTM!